cloudigrade/analyzer/tests/test_management_commands.py

```diff
@@ -0,0 +1,184 @@
+"""Collection of tests for Analyzer management commands."""
+import datetime
+import json
+import uuid
+from io import StringIO
+from unittest.mock import patch
+
+from django.core.management import call_command
+from django.test import TestCase
+from django.utils.translation import gettext as _
+
+from account.management.commands.add_account import aws
+from util.tests import helper
+
+
+class AnalyzeLogTest(TestCase):
+    """Analyze Log management command test case."""
+
+    def test_command_output_success(self):
+        """Test processing a CloudTrail log."""
+        out = StringIO()
+        mock_queue_url = 'https://sqs.queue.url'
+        mock_receipt_handle = str(uuid.uuid4())
+        mock_account_id = str(helper.generate_dummy_aws_account_id())
+        mock_instance_id = str(uuid.uuid4())
+        mock_region = 'us-east-1'
+        mock_event_type = 'RunInstances'
+        mock_occured_at = datetime.datetime.utcnow().isoformat()
+        mock_subnet = 'subnet-9000'
+        mock_ec2_ami_id = str(uuid.uuid4())
+        mock_instance_type = 't2.nano'
+
+        expected_instances = [(mock_account_id, mock_instance_id, mock_region)]
+        expected_instance_events = [
+            (mock_instance_id, 'power_on', mock_occured_at, mock_subnet,
+             mock_ec2_ami_id, mock_instance_type)
+        ]
+
+        expected_out = _('Found instances: {i} and events {e}'.format(
```

Comment:
I know this is still a problem all over our code, but if/when we _actually_ start gettexting our strings, we want to only gettext the unformatted parts. So, this would need to become:

```
expected_out = _('Found instances: {i} and events {e}').format(i=expected_instances, e=expected_instance_events)
```